modules/solutions/twitter-analytics/README.md

```diff
@@ -23,9 +23,8 @@ option.
     - On Windows:  `worker.bat --run`
     - On Linux/Mac OS:  `./worker.sh`
 
-+ Access the Business Rules Manager via one of the following URLs.
-    - `http://<SP_HOST>:<HTTP_PORT>/business-rules` **`(eg: http://0.0.0.0:9090/business-rules)`**
-    - `https://<SP_HOST>:<HTTPS_PORT>/business-rules` **`(eg: https://0.0.0.0:9443/business-rules)`**
++ Access the Business Rules Manager via the following URL.
+    - `https://<SP_HOST>:<HTTPS_PORT>/business-rules` **`(eg: https://0.0.0.0:9643/business-rules)`**
```

Comment:
Don't use 0.0.0.0 to access it should be localhost or the IP fix this in all places

Reply:
Addressed this in https://github.com/wso2/product-sp/pull/723